statsmodels/stats/knockoff_regeffects.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import statsmodels.api as sm
+
+
+class RegressionEffects(object):
+    """
+    Prorotype RegressionEffects class.
```

Comment:
typo Prototype
but maybe better is "Base class for regression effects used in RegressionFDR"